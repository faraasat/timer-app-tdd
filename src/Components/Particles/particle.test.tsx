import { shallow } from "enzyme";
import ParticlesComponent from "./particles.component";

describe("Particle Component Shallow Render", () => {
  const shallowRender = shallow(<ParticlesComponent />);

  it("should match snapshot", () => {
    expect(shallowRender.debug()).toMatchSnapshot();
  });

  it("should have id tsparticles", () => {
    expect(shallowRender.find("#tsparticles").length).toEqual(1);
  });
});
