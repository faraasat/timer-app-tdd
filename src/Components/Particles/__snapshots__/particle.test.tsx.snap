// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Particle Component Shallow Render should match snapshot 1`] = `"<h id=\\"tsparticles\\" options={{...}} width=\\"100%\\" height=\\"100%\\" style={{...}} />"`;
